configs/testdata/valkyrie_config.test.yml

```diff
@@ -22,6 +22,7 @@ pam: # player account management
   name: generic # check /pam-folder for available PAMs
   api_key: pam-api-key # api key to PAM
   url: "https://pam-url" # base url to PAM
+  settlement_type: "mixed" # Settlement type for the configured PAM, "mixed" or "gamewise" (default)
```

Comment:
Should we reference https://studio.evolutiongaming.com/api/evo-std-rest/docs/settlement-types.html somewhere to explain what these values mean?